src/org/robockets/steamworks/commands/Shoot.java

```diff
@@ -0,0 +1,37 @@
+package org.robockets.steamworks.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.robockets.commons.YAxisRelativeDirection;
+import org.robockets.steamworks.Robot;
+
+/**
+ * @author Jake Backer
+ */
+public class Shoot extends Command {
```

Comment:
It _should_ but I don't really want to make a command just to spin up the wheels